public/javascripts/PSMap/AddLabelsToMap.js

```diff
@@ -53,9 +53,19 @@ function AddLabelsToMap(map, labelData, params) {
             mapData.sortedLabels['Crosswalk'].map(l => l.length).reduce((acc, len) => acc + len, 0);
         document.getElementById('td-number-of-signals').innerHTML =
             mapData.sortedLabels['Signal'].map(l => l.length).reduce((acc, len) => acc + len, 0);
+
+        filterLabelLayers('incorrect', map, mapData, false);
     } else {
-        // For LabelMap. Set up the initial set of filters.
-        filterLabelLayers('incorrect', map, mapData);
+        // For LabelMap.
+        document.getElementById('map-legend-other').innerHTML =
```

Comment:
It looks like you accidentally reverted a change here when pulling in changes from develop!